contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -748,20 +938,25 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
             _execution.twapCooldownPeriod >= _incentive.incentivizedTwapCooldownPeriod,
             "TWAP cooldown must be greater than incentivized TWAP cooldown"
         );
-        require (
-            _execution.twapMaxTradeSize <= _incentive.incentivizedTwapMaxTradeSize,
-            "TWAP max trade size must be less than incentivized TWAP max trade size"
-        );
     }
 
+    /**
+     * Validate an ExchangeSettings struct when adding or updating an exchange. Does not validate that twapMaxTradeSize < incentivizedMaxTradeSize since
+     * it may be useful to disable exchanges for ripcord by setting incentivizedMaxTradeSize to 0.
+     */
+     function _validateExchangeSettings(ExchangeSettings memory _settings) internal pure {
+         require(_settings.exchangeLastTradeTimestamp == 0, "Last trade timestamp must be 0");
```

Comment:
This implies we will always reset the timestamp when we update an exchange. Is that what we want to do?

Reply:
Yeah think we can make it 100% clear by individually assigning the variables as you suggested above. It won't cause any weird state if exchange timestamp resets to 0 since the daily rebalance will read the global timestamp and we don't allow exchange setting updates when a rebalance is in progress (so won't affect TWAP cooldown period either)